### DISCUSSION TITLE
"Internal error" with no data

### DISCUSSION THREAD

**Discussion by rwckr** [bcb/jsonrpcserver#241] [Q&A]
I need to serve all Internal errors with no data in the response.  Is it recommended to remove the additional data from the response after the the dispatcher is done? Is there something I'm missing?

**Comment by bcb** [comment]
Yeah I think at the moment the only way is to remove it after the dispatch is finished.

**Reply by bcb** [reply]
In the previous version there was an option to remove this, maybe we should add it back.
